### PR TITLE
Add flower random spawner & random room showcase

### PR DESCRIPTION
**Pull request by Glamurio**
[Feature]
## About the PR 
This PR adds a new random spawner `/obj/random_item_spawner/flowers`.

It also adds a new random room to showcase it.

image

## Why's this needed? 
Having access to more random spawners allows for more varied, dynamic content.

### CODE REVIEW

**Comment by Glamurio** [comment]
I do not know how to fix that Beepsky bug, it's very confusing

**Comment by Glamurio** [comment]
I've spoken to several people and this is supposedly a Linter problem, not a problem in the actual code.

**Review by Azrun — CHANGES_REQUESTED** [review]
Now compiler angry.

File: code/obj/random_spawners.dm [hunk 1]
```diff
@@ -1897,3 +1897,63 @@
...
+		// Get a list of all flowers
+		items2spawn = concrete_typesof(/obj/item/clothing/head/flower) \
+		+ concrete_typesof(/obj/item/plant/flower) \
+		// Add some herbs that are basically flowers
+		+ list(/obj/item/plant/herb/poppy,
+		/obj/item/plant/herb/catnip,
+		/obj/item/plant/herb/hcordata
+		) \
+		// Exclude the non-natural ones
+		- list(/obj/item/plant/flower/rose/holorose)
```
[the remark this answers is not in the document]
Comment (Glamurio) [reply code/obj/random_spawners.dm]: Okay, I see what you're saying, I'll just make it several statements

**Pull request merged**